test/langtools/tools/javac/lint/LintWarningCategoryTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8346294
+ * @summary Verify LintWarnings.DanglingDocComment has a non-null associated LintCategory
+ * @modules
+ *      jdk.compiler/com.sun.tools.javac.resources
+ *      jdk.compiler/com.sun.tools.javac.util
+ * @run main LintWarningCategoryTest
+ */
+
+import com.sun.tools.javac.resources.CompilerProperties.LintWarnings;
+
+public class LintWarningCategoryTest {
```

Comment:
We don't strictly need this test, because build will fail now if any `lint:` label is invalid, so the test is redundant.

But I also wasn't sure if it was OK to not have any test at all.

Another option would be to split this change two separate bugs/PRs, but that seems like overkill.

(flips through OpenJDK developer guide...) Maybe the best answer would to remove the test and add the `noreg-build` label, as this can be considered a build fix.

Let's try that route... updated in 3f792a8b650.

Reply:
yeah, it's ok not to have a test, but you need some keyword, which you added.